### ISSUE TITLE
Cannot create tiles for United Kingdom

### ISSUE THREAD

**Issue by joouha** [valhalla/valhalla#2369]
Hello,

I've sucessfully build valhalla from master, but am unable to build tiles for the UK - running `valhalla_build_tiles` results in a `SIGSEGV` quite early on:

```
$ valhalla_build_tiles -c valhalla.json british-isles-latest.osm.pbf            

2020/05/13 16:09:11.794693 [INFO] Start stage = initialize End stage = cleanup
2020/05/13 16:09:12.700561 [INFO] Parsing files: british-isles-latest.osm.pbf
2020/05/13 16:09:12.718855 [INFO] Parsing ways...
2020/05/13 16:09:49.971836 [WARN] Max osmid exceeded bitset, resizing to fit id: 7393799674
2020/05/13 16:09:50.421018 [WARN] Max osmid exceeded bitset, resizing to fit id: 7393799674
2020/05/13 16:09:50.932413 [WARN] Max osmid exceeded bitset, resizing to fit id: 7495323946
2020/05/13 16:09:51.310215 [WARN] Max osmid exceeded bitset, resizing to fit id: 7486829833
2020/05/13 16:10:36.667032 [ERROR] Failed to execute lua function for basic tag processing.
fish: “valhalla_build_tiles  -c valhal…” terminated by signal SIGSEGV (Address boundary error)
```

Building tiles for Switzerland works fine.

Any ideas what the problem is?
Thanks

**Comment by kevinkreiser** [comment]
@joouha yep we know what it is. @zerebubuth is working on a fix at the moment. We should have a couple PRs addressing it shortly.

**Comment by zerebubuth** [comment]
Yup, I hope to have something soon. But if you're in a real hurry, then a quick work-around is to edit https://github.com/valhalla/valhalla/blob/1f0cf9fa493be9d44822d800eba6fab07cf41147/src/mjolnir/luatagtransform.cc#L100 and put a `return Tags{}` on the line after the `LOG_ERROR`. This will cause it to discard the way that's causing the problem, so you might get weird routes in that area, but the rest of the build should be able to complete OK.

**Comment by joouha** [comment]
That's really helpful, thanks

**Comment by jaegs** [comment]
Hey @zerebubuth is the underlying issue fixed? I am interested in patching the change if so. Thanks!

**Comment by zerebubuth** [comment]
Hi @jaegs,

Thanks for offering to help! :smile: 

However, I think this particular bug was fixed in PR #2371.
